### ISSUE TITLE
Using this script and getting „check for TSL2561 failed” on boot

### ISSUE THREAD

**Issue by BodoMinea** [nomath4u/crankshaft_tsl2591#1]
Hi, I have followed the wiki instructions (https://github.com/opencardev/crankshaft/wiki/TSL2591) on how to use a TSL2591 instead of a TSL2561 because that was far easier to get a hold of from a local store in my area. 

I haven't properly tested it in my car installation yet but from the get-go I have the following issue. With everything connected and done according to the wiki (also scanned for I2C addresses and I found 0x29 as a match, this being what the TSL2591 uses), I get the following message when Crankshaft-NG boots:

Check for TSL2561 failed. Device at 0x39 is missing!

I guess this check is hard-coded somewhere else. I tried searching with grep for mentions of TSL2561 or 0x39 in order to change to 0x29 but to no avail.

Also, `sudo i2cdetect -y 1` detects the TSL2591 just fine.

I have also ticked the „Show Lux value from tsl2561 on Main Window” in the TSL2561 tab of the Crankshaft settings, but that doesn't do anything for me.

Is there a way to also change the boot-check in order to not have warnings? It is nagging me a bit to see them every time.

Thanks

**Comment by nomath4u** [comment]
Hey. Unfortunately I was unable to figure out how to turn off those warnings as well. Checking the box for show lux values at the time of writing did work. Are you sure the script is running? I recommend sshing in and running the script while crankshaft is running on the main screen and see if it works then.

**Comment by BodoMinea** [comment]
Indeed, there was a problem running the script. I tried manually running it and it said `ModuleNotFoundError: No module named 'smbus'`.

Therefore, I typed in `sudo pip3 install smbus` and the Lux field appeared in the Crankshaft main menu GUI.

Running the script manually gives an error:

```
  File "/opt/crankshaft/service_lightsensor.py", line 56, in <module>
    if step <= get_var('TSL2561_DAYNIGHT_ON_STEP'):
NameError: name 'step' is not defined
```

But the lux values are changing in the UI. I see 220.7 under light and 4.7 when covering the sensor with my hand.

Script was ignoring values between levels 4 and 5. Fixed in PR #2
